WireMock/wwwroot/scripts/MatcherUI/PathMatcherUI.ts

```
@@ -0,0 +1,44 @@
+
+
+    // Renders Matcher row for the PATH Section
+    // and defining the behaviour of the user inputs
+    static renderPathMatcherUI(matcher: PathMatcher): HTMLElement {
+        let matcherRow = document.createElement("tr");
+        matcherRow.setAttribute("id", "matcher-" + matcher.Id);
+        matcherRow.appendChild(this.renderNameCell());
+        let rawMap = document.getElementById("rawMap-" + matcher.MappingGuid);
+        if (rawMap?.textContent) {
+            // Add the pattern cell and the behaviour
+            matcherRow.appendChild(this.renderPatternCell(matcher, (e) => {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)